examples/TCA9535_digitalWrite/TCA9535_digitalWrite.ino

```diff
@@ -22,7 +22,15 @@ void setup()
   Wire.begin();
   TCA.begin();
 
-  Wire.setClock(50);
+  Wire.setClock(50000);
+
+
+  Serial.println("Set pinMode1 OUTPUT");
+  for (int i = 0; i < 16; i++)
+  {
+    TCA.pinMode1(i, OUTPUT);
```

Comment:
would like to suggest adding `TCA.write1(i, LOW);` here so pins are cleared in initial state